mpyg321/mpyg321.py

```diff
@@ -123,7 +164,14 @@ def jump(self, pos):
     def handle_errors(self):
         """Handle errors encountered by the player"""
         output = self.player.readline().decode("utf-8")
-        raise PlayerError(output)
+
+        # Check error in list of errors
+        for message, error in mpg_errors.items():
```

Comment:
And here you would have:
```
for mpg_error in mpg_errors:
  if mpg_error.message in output:
    raise mpg_error.error(output)
```

Reply:
Done, though I should probably have used `mpg_error` rather than just `error`. Also because I chose to implement the errors as dictionaries I've used that notation instead, I hope that's okay.